back/db/migrate/20221208093437_add_banner_image_enabled_to_home_pages.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddBannerImageEnabledToHomePages < ActiveRecord::Migration[6.1]
+  def change
+    add_column :home_pages, :banner_image_enabled, :boolean, default: true, null: false
```

Comment:
AFAIR, we want to disable the image only for logged-in users, don't we? Maybe we should reflect it in the name.

Reply:
Good point!

`signed_in_banner_image_enabled` it is (will be).